src/input/LibretroDeviceInput.cpp

```diff
@@ -29,8 +29,9 @@ using namespace P8PLATFORM;
 #define LIBRETRO_JOYPAD_BUTTON_COUNT     16
 #define LIBRETRO_ANALOG_STICK_COUNT      2
 #define LIBRETRO_ACCELEROMETER_COUNT     1
-#define LIBRETRO_MOUSE_BUTTON_COUNT      7
-#define LIBRETRO_LIGHTGUN_BUTTON_COUNT   7
+#define LIBRETRO_MOUSE_BUTTON_COUNT      5
+#define LIBRETRO_MOUSE_BUTTON_START      RETRO_DEVICE_ID_MOUSE_LEFT
+#define LIBRETRO_LIGHTGUN_BUTTON_COUNT   5
```

Comment:
The libretro API uses 7 mouse buttons. I left the horizontal wheel up/down out of the mouse controller profile because I thought it'd be confusing in the GUI to ask users to map mouse buttons that most mice don't have. This means that mice with horizontal wheel buttons won't send those presses to the libretro core. What do you think we should do, simplify the GUI or provide support for horizontal wheel buttons?